### DISCUSSION TITLE
Adblocker ultimated telah menambahkan adblockid kedalam listvregional tolong di cek kak kayaknya itu gk berjalan maksimal
Daripada banyak aduan nanti

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1433] [General]
![20230710_222514_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/89764ffe-f3c3-4241-9c50-e10ef8e9eb22)

![20230710_222524_rmscr.jpg](https://github.com/realodix/AdBlockID/assets/114223791/915cf958-2413-48e4-ae57-38eaa45f2dd4)

**Comment by realodix** [comment]
Saya sudah melakukan pemeriksaan. 

Jika Anda melihat https://github.com/realodix/AdBlockID#manfaat-yang-anda-dapatkan
- No 2 sebagian tidak jalan
- No 3 sebagian besar tidak berjalan
- No 4 - 6 sepenuhnya tidak berjalan

Tapi walau bagaimanapun Saya sepenuhnya tidak bisa mengatur tentang masalah AdBlockID mau digunakan dimana saja, Saya hanya bisa sebatas menghimbau saja agar AdBlockID digunakan hanya di AdGuard dan uBO.
